db/migrate/20230125060440_create_comments.rb

```diff
@@ -0,0 +1,13 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.integer :authorid
+      t.integer :postid
+      t.text :text
+
+      t.timestamps
+    end
+    add_index :comments, :authorid
```

Comment:
- [ ] Again, the `user` model is referenced using foreign key in models as `AuthorId`, and `author` an alias of `user` hence, our migration query should be similar to:

```ruby
...
  def change
    add_reference :comments, :author, references: :users, null: false, foreign_key: {to_table: :users}
  end
...

```

or.. 

```ruby
  create_table :comments do |t|
    . . .
    t.references :author, foreign_key: { to_table: 'users' } 
    . . .
  end
```